Mathlib/NumberTheory/NumberField/InfinitePlace/Basic.lean

```diff
@@ -509,3 +511,77 @@ lemma isReal_infinitePlace : InfinitePlace.IsReal (infinitePlace) :=
   ⟨Rat.castHom ℂ, by ext; simp, rfl⟩
 
 end Rat
+
+namespace NumberField.InfinitePlace
+
+variable {K : Type*} [Field K] {v w : InfinitePlace K}
+
+/-- If `v` and `w` are infinite places of `K` and `v = w ^ t` for some `t` then `t = 1`. -/
+theorem eq_one_of_rpow_eq {t : ℝ} (h : (w ·) ^ t = v) : t = 1 := by
```

Comment:
Aren't we missing a lemma here, saying that the image of ℚ via infinite places coincides with the coercion from ℚ?